manifests/overlays/apps/base/elastic/elastic-app.yaml

```diff
@@ -0,0 +1,69 @@
+    <path d="M436.36 60.9101V50.5901H445.25V60.9101H436.36ZM436.36 128.48V68.4801H445.25V128.48H436.36Z" fill="#1C1E23"/>
+    <path d="M483.76 67.29C488.2 67.4795 492.622 67.9572 497 68.72L499.84 69.07L499.49 76.3C494.844 75.7114 490.172 75.3576 485.49 75.24C478.77 75.24 474.207 76.84 471.8 80.04C469.393 83.24 468.187 89.1667 468.18 97.82C468.18 106.487 469.307 112.513 471.56 115.9C473.813 119.287 478.513 120.987 485.66 121L499.66 119.94L500.02 127.29C494.57 128.213 489.065 128.768 483.54 128.95C474.3 128.95 467.92 126.577 464.4 121.83C460.88 117.083 459.12 109.083 459.12 97.83C459.12 86.57 461.017 78.6667 464.81 74.12C468.603 69.5734 474.92 67.2967 483.76 67.29Z" fill="#1C1E23"/>
+    </g>
+    <defs>
+    <clipPath id="clip0_15_21">
+    <rect width="500" height="171.38" fill="white"/>
+    </clipPath>
+    </defs>
+    </svg>
+  docsLink: https://www.elastic.co/platform
+  category: Self-managed
+  support: third party support
+  quickStart: ''
+  getStartedLink: https://www.elastic.co/guide/en/esre/current/index.html
+  csvName: elasticsearch-eck-operator-certified
+  getStartedMarkDown: >-
+    <p>Prototype and integrate with LLMs faster: Implement generative AI to build scalable, cost-effective applications across solutions using the Elastic Search AI Platform.</p>
+
+    <p>Make the most of your data with Elastic Search AI Platform's integrated solutions: search, security, and observability. Scale your generative AI capabilities and streamline operations with a flexible, secure, and production-ready platform.</p>
+
+    - Through features like chunking, [connectors](https://www.elastic.co/guide/en/enterprise-search/current/connectors.html), and web crawlers, Elastic ensures a seamless process for ingesting diverse datasets into the search 'layer'.
+
+    - Semantic search with ELSER, the built-in machine learning model, and use the [E5 embedding model](https://www.elastic.co/search-labs/blog/articles/multilingual-vector-search-e5-embedding-model) for multilingual vector search.  These models, and others,  address the need to ground LLMs, help prompt large language models like OpenAI, and pave the way for generative AI experiences.
+
+    - Elastic's API-first approach is enhanced by the Elastic Open Inference API, which streamlines code and manages inference for developers, providing flexibility to change models.
+
+    - Whether you use the built in ELSER model or integrate with embedding models from the ecosystem via Red Hat Openshift AI, Hugging Face, Cohere, OpenAI, or others for RAG workloads, a single, straightforward API call ensures clean code for managing hybrid inference deployment.
+
+    - With document and field-level security, Elasticsearch supports implementing permissions and entitlements that map to your organization’s role-based access control.
+
+    - Elasticsearch can be deployed where your apps are: on-prem, your cloud provider of choice, or air-gapped environments.
```

Comment:
Elastic Search is written in two ways... `Elastic Search` and `Elasticsearch`. Is this intentional?

Reply:
Yes  - this was provided by the vendor.  I've asked Kezia to review this on the cluster with them.